src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+
+  <div className="Person">
+    <h2 className="Person__name">
+      My name is
+      {person.name}
```

Comment:
Use interpolation here and fix word indent 
![image](https://github.com/mate-academy/react_person/assets/70009205/fe647a3e-487e-47bd-9852-0fe41d996f08)

```suggestion
      {`My name is ${person.name}`}
```